vignettes/CAMERA.Rnw

```diff
@@ -162,12 +163,12 @@ That's all for the simple sample approach. Please note that every method
 has additional parameters, that are not explicitly mentioned here. Also if your
 analysis doesn't need annotations, only a separation into groups, then simply
 stop after groupCorr. The grouping results are stored in the list
-\Rfunarg{object@pspectra}, which saves the peakindexes as elements for every group.
+\Rfunarg{object@pspectra}, which saves the peak indices as elements for every group.
 
 \begin{verbatim}
```

Comment:
quick thought, could we also change from `\verbatim` to actual inline code chunks ?